src/components/Modals/ConfirmationModal.jsx

```diff
@@ -79,36 +79,49 @@ const ConfirmationModal = ({
       aria-labelledby="dialog-title"
       aria-describedby="dialog-description"
       onClose={handleClose}
+      PaperProps={{
+        sx: {
+          borderRadius: '25px'
+        }
+      }}
```

Comment:
Nitpick, not too important, but If there's only one property in `sx`, think we could have this all collapse to one line.

Reply:
Do you mean making it just `sx={{ borderRadius: '25px' }}`? Because that doesn't work. It needs to use `PaperProps`.

Reply:
No, I mean `PaperProps={ sx={{ borderRadius: '25px' }} }`

Reply:
Think there's another one in the InactivityMessage component too. Can condense that into one line

Reply:
Oh, I got you. I think Prettier is set to do that by default. Anyway I've made it a single line manually.

Reply:
Ah okay. Think if prettier override the changes to the single line, then we'll just leave it alone then. If it sticks even after the formatting, change it to the one line.

Reply:
I'm pushing up the change. So far it's not fighting back on it, so it might be fine.